validation/util/test.go

```diff
@@ -138,3 +138,41 @@ func RuntimeInsideValidate(g *generate.Generator, f PreFunc) (err error) {
 	os.Stdout.Write(stdout)
 	return nil
 }
+
+// RuntimeOutsideValidate validate runtime outside a container.
+func RuntimeOutsideValidate(g *generate.Generator, f PreFunc) error {
+	bundleDir, err := PrepareBundle()
+	if err != nil {
+		return err
+	}
+
+	r, err := NewRuntime(RuntimeCommand, bundleDir)
+	if err != nil {
+		os.RemoveAll(bundleDir)
+		return err
+	}
+	defer r.Clean(true, true)
+	err = r.SetConfig(g)
+	if err != nil {
+		return err
+	}
+	err = fileutils.CopyFile("runtimetest", filepath.Join(r.BundleDir, "runtimetest"))
+	if err != nil {
+		return err
+	}
+
+	r.SetID(uuid.NewV4().String())
+	stderr, err := r.Create()
```

Comment:
no 'r.start'?

Reply:
> no 'r.start'?

What would you start?  The [bulk of the container is setup by the time `create` returns][1], and the runtime's dummy process is perfect for holding the container open while it is inspected from outside.  So I don't think we want a `start` call in `RuntimeOutsideValidate`.

[1]: https://github.com/opencontainers/runtime-spec/blame/v1.0.1/runtime.md#L103